src/patternfly/components/AppLauncher/app-launcher-menu-item-external-icon.hbs

```diff
@@ -1,6 +1,13 @@
-<i class="pf-c-app-launcher__menu-item-external-icon fas fa-external-link-alt{{#if app-launcher-menu-item-external-icon--modifier}} {{app-launcher-menu-item-external-icon--modifier}}{{/if}}"
-  aria-hidden="true"
+<span class="pf-c-app-launcher__menu-item-external-icon
+  {{#if app-launcher-menu-item-external-icon--modifier}} 
+    {{app-launcher-menu-item-external-icon--modifier}}
+  {{/if}}
   {{#if app-launcher-menu-item-external-icon--attribute}}
-	  {{{app-launcher-menu-item-external-icon--attribute}}}
-  {{/if}}></i>
+	    {{{app-launcher-menu-item-external-icon--attribute}}}
+  {{/if}}"
+>
```

Comment:
And can you move this up a line so it is after the closing `{{/if}}` block?

Reply:
I think I've got the last 3 things fixed